### DISCUSSION TITLE
NC AIO running on main domain, how do I switch to run on subdomain

### DISCUSSION THREAD

**Discussion by Corran2015** [nextcloud/all-in-one#4122] [Questions] [answered]
So while getting the AIO running, in the web interface where it asks for a domain, I just put in the domain, e.g. domainname.com. It accepted the domain and set up successfully.

If I wanted to somehow run it off a subdomain like cloud.domain.com, is it a matter of inputting cloud.domain.com on the AIO set up interface, or is that something I work on the DNS records?  Also, if I have it running now, is there a way to change it to run off the subdomain, or will I need to restart the setup?

DNS records set up with the domain registrar for external IP like:
A @ x.x.x.x
A * x.x.x.x

with x.x.x.x being my external IP.

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-change-the-domain
